modules/staking_pool/src/lib.rs

```diff
@@ -72,14 +111,31 @@ decl_storage! {
 		pub CurrentEra get(fn current_era): EraIndex;
 
 		pub NextEraUnbond get(fn next_era_unbond): (Balance, Balance);
-		pub Unbonding get(fn unbonding): map hasher(twox_64_concat) EraIndex => (Balance, Balance); // (value, claimed), value - claimed = unbond to free
+		pub Unbonding get(fn unbonding): map hasher(twox_64_concat) EraIndex => (Balance, Balance, Balance); // (unbounding, total_claimed, initial_total_claimed)
 
 		pub ClaimedUnbond get(fn claimed_unbond): double_map hasher(twox_64_concat) T::AccountId, hasher(twox_64_concat) EraIndex => Balance;
 		pub TotalClaimedUnbonded get(fn total_claimed_unbonded): Balance;
 
 		pub TotalBonded get(fn total_bonded): Balance;
 		pub UnbondingToFree get(fn unbonding_to_free): Balance;
 		pub FreeUnbonded get(fn free_unbonded): Balance;
+
+		pub GlobalParams get(fn global_params): Params;
```

Comment:
```suggestion
		pub GlobalParams get(fn global_params) config(): Params;
```
Just need to add `#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]` for `struct Params`